seqr/utils/elasticsearch/es_search.py

```diff
@@ -824,6 +825,16 @@ def _parse_genome_versions(self, result, index_name, hit):
             'liftedOverPos': lifted_over_pos,
         })
 
+    def _parse_transloc_xstop(self, result, index_name):
+        xstop = result.pop(XSTOP_FIELD, None)
+        if xstop:
+            endChrom, end = get_chrom_pos(xstop)
+            if endChrom != result['chrom'] or end != result['end']:
```

Comment:
endChrom should only be set if it is different from chrom, so this logic does not work. This should be 2 seperate if statements:
```
if end_chrom != result['chrom']:
  result['endChrom'] = end_chrom
if end != result['end']:
  result['end'] = end
```

Reply:
The logic `endChrom != result['chrom'] or end != result['end']` doesn't work in the tests. To be honest, I don't know why. I think the chrom:end in `xstop` is different from that in the result only when there is CHR2 and END2 in the pipeline input. So we can use the condition to detect CHR2 and END2.
But, I'll go with your original suggestion and assign the `endChrom` and `end` when the `end_chrom` is different from `result['chrom']`. The only issue is that we can't display the second coordination if the translocation target is in the same chromosome.
I'm hesitating to check and assign `end` separately. It could be wrong for some CPX variants.